src/lib/content.d.ts

```diff
@@ -234,9 +234,8 @@ interface VideoVimeoBlockElement {
     height: number;
     width: number;
     caption?: string;
-    credit: string;
-    title: string;
-    description: string;
```

Comment:
We're losing `description` here, is that intentional?

Reply:
yep